tests/integration/base.py

```diff
@@ -143,43 +145,70 @@ class DBTIntegrationTest(unittest.TestCase):
     prefix = f'test{_runtime}{_randint:04}'
     setup_alternate_db = False
 
-    def databricks_cluster_profile(self):
-        return {
+    def _build_databricks_cluster_profile(
+        self,
+        http_path: str,
+        catalog: Optional[str] = None,
+        session_properties: Dict[str, str] = None,
+    ):
+        profile = {
             'config': {
                 'send_anonymous_usage_stats': False
             },
             'test': {
                 'outputs': {
-                    'dbsql': {
+                    'dev': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
-                        'http_path': os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH'),
+                        'http_path': http_path,
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
                 },
-                'target': 'dbsql'
+                'target': 'dev'
             }
         }
+        if catalog is not None:
+            # TODO: catalog should be set as 'catalog' or 'database'
+            #       instead of using 'session_properties'
+            # profile['test']['outputs']['dev']['catalog'] = catalog
+            if session_properties is not None:
+                session_properties['databricks.catalog'] = catalog
+            else:
+                session_properties = {'databricks.catalog': catalog}
+        if session_properties is not None:
+            profile['test']['outputs']['dev']['session_properties'] = session_properties
+        return profile
+
+    def databricks_cluster_profile(self):
+        return self._build_databricks_cluster_profile(
+            http_path=os.getenv(
+                "DBT_DATABRICKS_CLUSTER_HTTP_PATH", os.getenv("DBT_DATABRICKS_HTTP_PATH")
+            ),
+        )
+
+    def databricks_uc_cluster_profile(self):
+        return self._build_databricks_cluster_profile(
+            http_path=os.getenv(
```

Comment:
I see this appears in multiple places. How about let's define them in a separate `config.py` file for all the configurations? For example
```python
# tests/integration/config.py

DBT_DATABRICKS_DEFAULT_HTTP_PATH = os.getenv("DBT_DATABRICKS_HTTP_PATH")
# A databricks cluster HTTP path with Unity Catalog enabled.
DBT_DATABRICKS_UC_CLUSTER_HTTP_PATH = os.getenv("DBT_DATABRICKS_UC_CLUSTER_HTTP_PATH", DBT_DATABRICKS_DEFAULT_HTTP_PATH)
...
```

Reply:
I'm afraid I don't think it's worth doing it.
We will see `os.getenv( ... )` anyway and I don't want to separate the test configurations here and there.
Actually there are already many places we need to modify to add a profile.

Maybe I can modify a bit more to deduplicate code. Let me see.

Reply:
Hmm, we could do something like:

```diff
@@ -147,7 +147,7 @@ class DBTIntegrationTest(unittest.TestCase):

     def _build_databricks_cluster_profile(
         self,
-        http_path: str,
+        http_path: Optional[str],
         catalog: Optional[str] = None,
         session_properties: Dict[str, str] = None,
     ):
@@ -160,7 +160,11 @@ class DBTIntegrationTest(unittest.TestCase):
                     'dev': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
-                        'http_path': http_path,
+                        'http_path': (
+                            http_path
+                            if http_path is not None
+                            else os.getenv('DBT_DATABRICKS_HTTP_PATH')
+                        ),
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
@@ -182,9 +186,7 @@ class DBTIntegrationTest(unittest.TestCase):

     def databricks_cluster_profile(self):
         return self._build_databricks_cluster_profile(
-            http_path=os.getenv(
-                "DBT_DATABRICKS_CLUSTER_HTTP_PATH", os.getenv("DBT_DATABRICKS_HTTP_PATH")
-            ),
+            http_path=os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH")
         )

     def databricks_uc_cluster_profile(self):
```

but I'd not prefer this because it's not very clear that `http_path` will be delegated to `os.getenv('DBT_DATABRICKS_HTTP_PATH')` when the env var doesn't exist.

WDYT?